### ISSUE TITLE
Vortex 1.12.3 auto disables SFBGS004.ESM after exiting Starfield and Vortex.

### ISSUE THREAD

**Issue by darth-crunchus** [Nexus-Mods/Vortex#16391]
**Describe the bug**
SFBGS004.ESM (The latest esm for Starfield is auto disabled after exiting the game and Vortex.

**To Reproduce**
1. Launch Vortex and ensure that you are managing Starfield
2. If Necessary, go to Plugins and re-enable SFBGS004.ESM and auto-sort.
3. Close Vortex.
4. Start Starfield (doesn't matter if from Steam, Vortex or SFSE Loader), and play a little bit.
5. Re-open Vortex.  If you have mods that require SFBGS004.ESM, you'll receive a "Missing Masters" notification.  If not, got to Plugins tab and see that SFBGS004.ESM is inexplicably disabled.

**Expected behavior**
I expected a game master file to stay enabled.
**Screenshots**
![vortex problem](https://github.com/user-attachments/assets/43548abd-9ede-4187-9fe6-ae40a62384f5)

**Platform (please complete the following information):**
 - OS: Windows 11 Pro 23H2 (22631.4112)
 - Vortex Version 1.12.3
**Additional context**

**Comment by ChemGuy1611** [comment]
LOOT needs to be updated to 0.23.1 to fix this. An issue exists for this already. 

https://github.com/Nexus-Mods/Vortex/issues/16344 

https://loot.readthedocs.io/en/latest/app/changelog.html

**Comment by darth-crunchus** [comment]
> LOOT needs to be updated to 0.23.1 to fix this. An issue exists for this already.
> 
> #16344
> 
> https://loot.readthedocs.io/en/latest/app/changelog.html

Wait, so, Nexus integrates LOOT into Vortex, but it's OUR job to update it?

**Comment by ChemGuy1611** [comment]
No, Nexus Mods team needs to update the LOOT version integrated in the plugin management extension in Vortex.

**Comment by darth-crunchus** [comment]
Ahhhhhh, I gotcha now.

**Comment by insomnious** [comment]
Working on this as we speak, should get an update out tomorrow (Wednesday)
